### PR TITLE
docs(colorgradient):revamp overview

### PR DESCRIPTION
**Pull request by ntacheva**

### CODE REVIEW

File: components/colorgradient/overview.md [hunk 1]
```diff
@@ -62,6 +46,26 @@ The ColorGradient accepts values by the application code in the following format
...
+## ColorGradient Parameters
+
+The Blazor ColorGradient provides various parameters to configure the component. Also check the [ColorGradient public API](/blazor-ui/api/Telerik.Blazor.Components.TelerikColorGradient).
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type and Default Value | Description |
+| --- | --- | --- |
+| `Class` | `string` | The class that will be rendered on the main wrapping elemnet of the component (`<div class="k-colorgradient">`). | 
+| `Enabled`| `bool` | Whether the ColorGradient is enabled. |
```
Comment (yordan-mitev) [inline components/colorgradient/overview.md:58]:
```suggestion
| `Enabled`| `bool` | Defines whether the ColorGradient is enabled. |
```

File: components/colorgradient/overview.md [hunk 2]
```diff
@@ -62,6 +46,26 @@ The ColorGradient accepts values by the application code in the following format
...
+| `Value` | `string` | The ColorGradient value. Accepts [several different color formats](#supported-value-formats). Supports two-way binding.
+| `ValueFormat` | `ColorFormat` enum <br/> (`ColorFormat.Rgb`)| The color format, which the **component will return** in the application code.
```
Comment (yordan-mitev) [inline components/colorgradient/overview.md:60]:
```suggestion
| `ValueFormat` | `ColorFormat` enum <br/> (`ColorFormat.Rgb`)| The color format which the component will return in the application code.
```

File: components/colorgradient/overview.md [hunk 3]
```diff
@@ -62,6 +46,26 @@ The ColorGradient accepts values by the application code in the following format
...
+| `Format` | `ColorFormat` enum <br/> (`ColorFormat.Rgb`) | The value format, which the users **sees initially**. Supports two-way binding. The `Rgb` input format allows changing the textbox values with the Up/Down arrow keys.
```
Comment (yordan-mitev) [inline components/colorgradient/overview.md:61]:
```suggestion
| `Format` | `ColorFormat` enum <br/> (`ColorFormat.Rgb`) | The value format which the users sees initially. Supports two-way binding. The `Rgb` input format allows changing the textbox values with the Up/Down arrow keys.
```

File: components/colorgradient/overview.md [hunk 4]
```diff
@@ -62,6 +46,26 @@ The ColorGradient accepts values by the application code in the following format
...
+| `Formats` | `IEnumerable<ColorFormat>` | The available color formats, which the user **can see, toggle and edit** by typing. Both `Hex` and `Rgb` formats are enabled by default and the user can switch between them with a button.
```
Comment (yordan-mitev) [inline components/colorgradient/overview.md:62]:
```suggestion
| `Formats` | `IEnumerable<ColorFormat>` | The available color formats which the user can see, toggle, and edit by typing. Both `Hex` and `Rgb` formats are enabled by default, and the user can switch between them with a button.
```

File: components/colorgradient/overview.md [hunk 5]
```diff
@@ -62,6 +46,26 @@ The ColorGradient accepts values by the application code in the following format
 
 Color keywords are not supported. If this is the preferred use case scenario, consider the [ColorPalette component]({%slug colorpalette-overview%}).
 
+## ColorGradient Parameters
+
+The Blazor ColorGradient provides various parameters to configure the component. Also check the [ColorGradient public API](/blazor-ui/api/Telerik.Blazor.Components.TelerikColorGradient).
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type and Default Value | Description |
```
Comment (dimodi) [inline components/colorgradient/overview.md:55]: Sort the parameters alphabetically.

**Review by dimodi — APPROVED** [review]

**Pull request merged**